src/refresh/vkpt/shader/global_ubo.h

```diff
@@ -266,8 +267,7 @@ BEGIN_SHADER_STRUCT( ModelInstance )
 	int iqm_matrix_offset_curr_frame;
 	int iqm_matrix_offset_prev_frame;
 
-	int frame;
-	float alpha;
+	uint frame_and_alpha;
```

Comment:
I was a bit surprised to find that 'frame' is in the high 16 bits and 'alpha' is in the low 16 bits. I usually name combined variables like that in the opposite order, so `alpha_and_frame`, see e.g. `emissive_and_alpha` in `bsp_mesh.c`. Please rename and (ideally) add a comment explaining the packing.

Reply:
Somehow that pattern totally went past me. Addressed.

Reply:
No worries, and thanks for fixing it!